faw/pdf-observatory/faw_analysis_set_parse.py

```diff
@@ -408,9 +499,22 @@ def coll_resolver(name):
         col_obs.update_one({'_id': doc_id}, {'$set': {'idle_complete': True}})
 
 
-def _as_run_tool(col_dst, fpath, fpath_tool_name, parser_inv_name,
-        parser_tool_version, parser_cfg, api_info, *, timeout_default):
-    """Re-implementation of pdf-etl-tool.
+def as_run_tool(
```

Comment:
Now public, since it can be invoked when running file plugins that require artifacts.

Also now safe to run outside of dask.